CalGads/CalGads/Model/keycomposition.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  keycomposition.swift
+//  CalGads
+//
+//  Created by Gadiel Paiz on 1/31/24.
+//
+
+import Foundation
+
+struct keyComposition {
+    let funtions: [KeyFuntion] = [.primary, .secondary, .tertiary, .quaternary, .none]
```

Comment:
remove de value. In the model we write only definition like "let functions: [KeyFuntion]